### PR TITLE
Release v0.1.1-alpha.19

### PR DESCRIPTION
**Pull request by soapraj**
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: microsite/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0",
+  "version": "0.1.1-alpha.19",
```
Comment (freben) [inline microsite/package.json:2]: will this have any particular effect? like, refusing to deploy because it's a downgrade or something :)

**Review by freben — APPROVED** [review]

File: microsite/package.json (hunk 1 above)
Comment (soapraj) [reply microsite/package.json:2]: I don't think so. It's a private package and won't be published.

**Comment by Fox32** [comment]
⚠️  We noticed that this will cause problems while publishing: https://github.com/spotify/backstage/blob/master/plugins/api-docs/package.json#L7

Not sure how that happened on our side? Did we made a mistake while copying or using the create plugin script?

**Comment by Fox32** [comment]
> Not sure how that happened on our side? Did we made a mistake while copying or using the create plugin script?

Nevermind, it's the default from create plugin... 🙄  https://github.com/spotify/backstage/blob/master/packages/cli/templates/default-plugin/package.json.hbs#L7

**Comment by freben** [comment]
Rebased to rebuild on top of #2050

**Comment by freben** [comment]
@Fox32 we want to make it an option, see #615

**Comment by freben** [comment]
@dtuite it currently won't build for some reason because of backstage-plugin-travis-ci it seems :)

@Rugvip hmm i wonder if it's related to the filtering on a `@backstage` dep prefix?

**Comment by dtuite** [comment]
@freben Let me know if you want me to do something. If you're in a rush, maybe remove the plugin and I will take a look over the weekend. Here's the commit where I added it. https://github.com/spotify/backstage/pull/1892/commits

There haven't been any commits to [the plugin](https://github.com/RoadieHQ/backstage-plugin-travis-ci) or releases in 11 days.

**Comment by soapraj** [comment]
@dtuite We think the reason could be because travis-ci plugin is pinned to a specific version

This change should fix that https://github.com/RoadieHQ/backstage-plugin-travis-ci/pull/3 

When we bump versions for backstage packages we bump the versions in all the plugins that depend on it as well and since this was an external repo we can't make the same change.

**Pull request merged**